### PR TITLE
Dockerfile and Makefile to login, build and push containers to ECR

### PR DESCRIPTION
**Pull request by frdvo**
Dockerfile and Makefile to login, build and push containers to ECR

### CODE REVIEW

File: docker/Makefile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+TAG ?= $(shell git rev-parse --short HEAD)
+REPO_URL ?= $(shell cd ../terraform&&terraform output -json ecr-module | jq .ecr | jq -r .repository_url)
+#CONTAINER_NAME ?= wppj3
+
+.PHONY: login
+login:
+	@echo "Retrieving an authentication token and authenticate your Docker client to your registry"
+	aws ecr get-login-password --region ap-southeast-2 | docker login --username AWS --password-stdin ${REPO_URL}
```
Comment (jayanath) [inline docker/Makefile:8]: --username AWS? Can we have this as a parameter?

File: docker/Makefile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+TAG ?= $(shell git rev-parse --short HEAD)
```
Comment (jayanath) [inline docker/Makefile:1]: Not sure if we should tag the image with a commit ID.

**Review by jayanath — APPROVED** [review]
Good work 🌠 . Have few comments.

File: docker/Makefile (hunk 1 above)
Comment (frdvo) [reply docker/Makefile:8]: This AWS CLI command provides authentication to local docker-cli.  You can these commands visiting your ECR repository on AWS Portal.

File: docker/Makefile (hunk 2 above)
Comment (frdvo) [reply docker/Makefile:1]: this is a requirement from issue #7

**Pull request merged**